app/src/main/java/ie/macinnes/tvheadend/TvContractUtils.java

```diff
@@ -62,6 +62,24 @@ public static long getChannelId(Context context, int channelId) {
         return INVALID_CHANNEL_ID;
     }
 
+    public static String getChannelName(Context context, int channelId) {
+        ContentResolver resolver = context.getContentResolver();
+
+        Uri channelsUri = TvContract.buildChannelsUriForInput(TvContractUtils.getInputId());
+
+        String[] projection = {TvContract.Channels.COLUMN_ORIGINAL_NETWORK_ID, TvContract.Channels.COLUMN_DISPLAY_NAME};
```

Comment:
The value being passed in here is the _ID column (aka android ID), while we keep the tvhChannelId in COLUMN_ORIGINAL_NETWORK_ID?